10-Types-of-contributions.md

```diff
@@ -0,0 +1,228 @@
+
+### How to Contribute to Documentation
+
+*If you're interested in contributing to documentation, here are a few tips:*
+
+- **Read the documentation**: This will help you understand how the product is used.
+  
+- **Find a task that's a good fit for your skills and interests**: If you're not sure where to start, ask the project's maintainers for suggestions.
+  
+- **Use the product**: Gaining user insight will help you see which information is needed or not useful in the docs.
+  
+- **Be patient and persistent**: It takes time to write good documentation. Don't be afraid to ask for help if you get stuck.
+  
+
+#### Here are some specific examples of documentation contributions:
+
+
+- Writing a README file that explains how to install and use the project
+  
+
+*Example*
+
+[Create a Bootstrap Example ](https://github.com/Mirrorful/mirrorful/pull/321)
+
+- Creating a tutorial that walks users through the project's features
+
+*Example*
+
+[Adding chart](https://github.com/rupali-codes/LinksHub/pull/1632#event-10288094144)
+
+- Updating the project's documentation to reflect new changes
+
+*Example*
+
+[Add Dev Cards to Getting Started](https://github.com/open-sauced/docs/issues/156)
+
+- Translating the project's documentation into other languages
+  
+*Example*
+
+[Translate the course into another language](https://github.com/open-sauced/intro/issues/16)
+
+- Fixing typos and grammatical errors in the documentation
+
+*Example*
+
+[Adding information about possessives
+](https://github.com/StyleGuides/WritingStyleGuide/pull/519)
+
+- Improving the formatting of the documentation
+
+*Example*
+
+  [Incorrect Link Destination in Documentation](https://github.com/open-sauced/docs/pull/144)
+  
+- Adding missing information to the documentation
+
+*Example*
+
+[Add More Information to Use This Template in the README
+](https://github.com/open-sauced/100-days-of-oss-template/pull/8)
+
+Contributing to documentation is a great way to help open-source projects and make them more accessible to everyone. If you have the skills and the interest, I encourage you to get involved!
+
+### Graphic Design
+
+Graphic design is a type of open-source contribution that involves creating visual assets, such as logos, icons, illustrations, website designs, and other graphical elements, for open-source projects. These visual assets play a significant role in enhancing the overall user experience and branding of the project. Graphic design contributions are important because they can make open-source software more appealing, user-friendly, and recognizable.
+
+ ### Different graphic design open-source contributions 
+ 
+ - **Logo and Branding**: Designing a unique and memorable logo for an open-source project can help establish its identity and brand. A well-designed logo can make the project instantly recognizable and give it a professional image.
+
+*Example*
+
+[Insights on social card generation](https://github.com/open-sauced/opengraph/pull/51)
```

Comment:
I don't think this matches up with logo and branding. It's ok if you can't find an example. That's tricky.